README.md

```diff
@@ -4,7 +4,9 @@
 
 -   `BOT_TOKEN` - your Discord bot token
 -   `CLIENT_ID` - your Discord bot application ID (used for the invite link) (optional)
+-   `BOT_PREFIX` - your preferred bot prefix if not using slash commands (default is `2!`)
```

Comment:
Please sort this alphabetically.

Reply:
I put them in order logically. Up to you of you want to sort.